templates/repo/actions/list.tmpl

```diff
@@ -57,6 +61,18 @@
 							{{end}}
 						</div>
 					</div>
+
+					{{if .AllowDisableOrEnableWorkflow}}
+						<button class="ui small compact jump dropdown icon btn interact-bg gt-p-3">
```

Comment:
`ui dropdown button` is good enough, I think there is no need to use `btn interact-bg gt-p-3`.

Reply:
Such a button would not fit visually:

<img width="241" alt="image" src="https://github.com/go-gitea/gitea/assets/115237/37cbf3d0-4de0-40c1-8d9f-882a235c92ba">

Reply:
I removed some unnecessary classes in https://github.com/go-gitea/gitea/pull/26413/commits/b2a5c22c88be5de5675a9460375ac520f648b87b, I think this is the minimum to achieve the desired look.